stripe/resource.py

```diff
@@ -525,6 +525,15 @@ def mark_as_safe(self, idempotency_key=None):
         return self
 
 
+class Dispute(CreateableAPIResource, ListableAPIResource,
+              UpdateableAPIResource):
+    def close(self, idempotency_key=None):
```

Comment:
Other classes in the file have a line break before the first method. Seems like PEP8 doesn't care.